activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -164,7 +164,7 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-donald-e-knuth">Donald E. Knuth</a>
-  def parameterize(sep = '-')
+  def parameterize(sep = '-'.freeze)
```

Comment:
@bquorning That said, I remember a way I did that in a different project. That was by using constants in cases like these. For example, `def parameterize(sep = DASH_TOKEN)`. That way, the `object_id`s will be same no matter where this token gets used.